test/specs/manifestBuilder.e2e.ts

```diff
@@ -0,0 +1,227 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import path from 'path';
+import { step } from 'mocha-steps';
+import { TestSetup } from '../testSetup';
+import { TextEditor } from 'wdio-vscode-service';
+import * as utilities from '../utilities';
+
+describe('Manifest Builder', async () => {
+  let testSetup: TestSetup;
+  let projectName: string;
+  const fiveMinutes = 5 * 60;
+
+  step('Set up the testing environment', async () => {
+    utilities.log(`ManifestBuilder - Set up the testing environment`);
+    testSetup = new TestSetup('ManifestBuilder', false);
+    await testSetup.setUp();
+    projectName = testSetup.tempProjectName.toUpperCase();
+
+    utilities.log(
+      `${testSetup.testSuiteSuffixName} - calling createCustomObjects()`
+    );
+    utilities.createCustomObjects(testSetup);
+
+    utilities.log(`${testSetup.testSuiteSuffixName} - creating manifest file`);
+
+    const workbench = await browser.getWorkbench();
+
+    // Using the Command palette, run File: New File...
+    const inputBox = await utilities.runCommandFromCommandPrompt(
+      workbench,
+      'Create: New File...',
+      1
+    );
+
+    // Set the name of the new manifest file
+    const filePath = path.join('/manifest/manifest.xml');
+    await inputBox.setText(filePath);
+    await inputBox.confirm();
```

Comment:
Could you add comments to each of the calls to `inputBox.confirm()`?  I'm sure they are needed, but as a reader of the code, I don't know what each one is doing.